scipy/interpolate/fitpack2.py

```diff
@@ -425,17 +426,17 @@ def integral(self, a, b):
         return dfitpack.splint(*(self._eval_args+(a, b)))
 
     def derivatives(self, x):
-        """ Return all derivatives of the spline at the point x.
+        """ Return all derivatives of the spline at the point `x`.
 
         Parameters
         ----------
         x : float
-            The point to evaluate the derivatives at.
+            The point to evaluate the derivatives at `x`.
 
         Returns
         -------
-        der : ndarray, shape(k+1,)
-            Derivatives of the orders 0 to k.
+        d : ndarray, shape(k+1,)
```

Comment:
Avoid unnecessary changes like this one. Here it would not change anything so please revert such changes.

Reply:
I agree with the part `x`. In case of `der` vs. `d`, the function `derivatives` returns just d and not `der`. I personally find it a little missleading

Reply:
Still it should be coherent with the code here. As I said, we should not change this as it would not do anything.